odd-platform-ui/src/components/shared/AppGraph/AppGraphNode/AppGraphNode.tsx

```diff
@@ -350,15 +350,15 @@ const AppGraphNode: React.FC<AppGraphNodeProps> = ({
               </>
             )}
         </g>
-        {data.types?.map((type, i) => (
+        {data.entityClasses?.map((entityClass, i) => (
           <g
-            key={type.id}
+            key={entityClass.id}
             transform={`translate(${
               titleLayout.x + i * (typeLayout.width + typeLayout.mx)
             },${nodeSize.y - typeLayout.my - typeLayout.height})`}
           >
             <TypeContainer
-              $typeName={type.name}
+              $typeName={entityClass.name}
               width={typeLayout.width}
               height={typeLayout.height}
             />
```

Comment:
Now it's `EntityClassContainer` with `$entityClassName` prop

Reply:
done